app/picture.py

```diff
@@ -22,26 +24,47 @@ def __init__(self, img_path: str) -> None:
         self.tile_order: list[int] = []
         self.scramble_type = None
 
-    def save(self, img_path: str) -> None:
```

Comment:
Was removing `save` intentional?

Reply:
Yes, I think in the final code it's not used anymore, or am I wrong?
I mean, we can still keep it, but in the end I didn't understand how it actually worked anymore. Or that it actually didn't work for circular scrambled pictures